### PR TITLE
silly ningen added old one

### PR DESCRIPTION
**Pull request by osmos1**
added string so you know, if and else if doesnt just stand there doin nothin

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -20,15 +20,15 @@ static void Main(string[] args)
...
-                    Console.WriteLine("That's nice.");
+                    Console.WriteLine("I don't care.");
 
                 else if (answer == "Bad")
-                    Console.WriteLine("That's too bad.");
+                    Console.WriteLine("I don't care.");
 
                 else
-                    Console.ReadLine();
+                    Console.WriteLine("Are youretarded? Answer my question.");
```
Comment (Couls) [inline Program.cs:29]: missing space

File: Program.cs [hunk 2]
```diff
@@ -54,14 +54,16 @@ static void Main(string[] args)
...
-                    Console.WriteLine("...");
+                string answer2 = Console.ReadLine();
+                {
+                    if (answer2 == "Y")
+                        Console.WriteLine("...");
 
-                else if (answer == "N")
-                    Console.WriteLine("...");
-               }
+                    else if (answer2 == "N")
```
Comment (Couls) [inline Program.cs:63]: why renaming string?

**Review by Couls — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 2 above)
Comment (Couls) [reply Program.cs:63]: also outdated PR :( can't merge

Comment (osmos1) [reply Program.cs:63]: thought it would break it but i guess not

Comment (Couls) [reply Program.cs:63]: we'll test with debugger later on
think we gotta close this PR tho

**Pull request closed without merging**